src/script/ceph-backport.sh

```diff
@@ -49,6 +48,25 @@
 # filesystem, etc.). Without correct values for these four variables, this
 # script will not work!
 #
+# Instructions for use
+# --------------------
+#
+# Assumes you have forked ceph/ceph.git, cloned your fork, and are running the
+# script in the local clone!
+#
+# With this script, backporting workflow for backport issue
+# http://tracker.ceph.com/issues/19206 (a jewel backport)
+# becomes something like this:
+#
+# git remote add ceph http://github.com/ceph/ceph.git
+# git fetch ceph
+# git checkout -b wip-19206-jewel ceph/jewel
+# git cherry-pick -x ...
```

Comment:
For cherry-picking I have a trick that I use locally, 
```
git fetch <upstream> pull/<prr-num>/head:pr-<prnum>
git cherry-pick -x pr-<pr-num>~<num-commits>..pr-<pr-num>

```
Adding following to somewhere bash can find  can make things easier

```sh
prfetch() {
    git fetch upstream pull/${1}/head:pr-${1}
```

Reply:
ok, added this material to http://tracker.ceph.com/projects/ceph-releases/wiki/HOWTO_backport_commits (under "Notes on cherry-picking")

Reply:
thanks!